vncdotool/loggingproxy.py

```diff
@@ -31,19 +32,20 @@ def connectionMade(self):
         Protocol.connectionMade(self)
         self.transport.setTcpNoDelay(True)
 
-        self.buffer = ''
+        self.buffer = b''
```

Comment:
Did you test this on both Python 2 and 3?

Reply:
I have tested with version 2.7.16 and 3.7.3

Reply:
Changed by 756c5f89